test/e2e/flowaggregator_test.go

```diff
@@ -213,6 +216,73 @@ func testFlowAggregatorAntctl(t *testing.T, data *TestData) {
 			}
 		})
 	}
+
+	t.Run("antctl get flowrecords -o json", func(t *testing.T) {
+		var cmdStr, srcIP, dstIP string
+		// trigger a flow with iperf
+		if !isIPv6 {
+			srcIP = podAIPs.ipv4.String()
+			dstIP = podBIPs.ipv4.String()
+			cmdStr = fmt.Sprintf("iperf3 -c %s -t %d", dstIP, 6)
+		} else {
+			srcIP = podAIPs.ipv6.String()
+			dstIP = podBIPs.ipv6.String()
+			cmdStr = fmt.Sprintf("iperf3 -6 -c %s -t %d", dstIP, 6)
+		}
+		stdout, _, err := data.runCommandFromPod(testNamespace, "perftest-a", "perftool", []string{"bash", "-c", cmdStr})
+		if err != nil {
+			t.Errorf("Error when running iperf3 client: %v", err)
+		}
+		_, srcPort, dstPort := getBandwidthAndPort(stdout)
+
+		// run antctl command on flow aggregator to get flow records
+		var args []string
+		command := []string{"get", "flowrecords", "-o", "json", "--srcip", srcIP, "--srcport", srcPort}
+		if testOptions.enableCoverage {
+			antctlCovArgs := antctlCoverageArgs("antctl-coverage")
+			args = append(antctlCovArgs, command...)
+		} else {
+			args = append([]string{"antctl"}, command...)
```

Comment:
Looks like variable `args` is storing the command while `command` is storing the arguments of antctl command?

Reply:
Exchanged the variable name.